main.cpp

```diff
@@ -402,7 +410,37 @@ Thing 6) Mixing Room
     2) send files to server
     3) connect to DSP racks
  */
-
+struct mixRoom
+{
+  mixRoom();
+
+  std::string roomName;
+  void sizeOfRoom(float height, float width);
+  float sizeOfSpeaker;
+  double roomAcousticValue;
+  float amountOfPowerConsumed;
+
+  struct speakers
+  {
+    speakers();
+
+    std::string brand = "Dynaudio";
+    std::string model = "LYD5";
+    double speakerWright = 5.7;
+    double crossoverFreq = 5.2;
+    int maxSpl = 106;
+
+    void insertPower();
+    void settingSensivity(int dbSettings1 = 6, int dbSettings2 = -6);
+    void insertSoundCard(int inputNumber = 3);
+  };
+
+  void allowsCommunication(speakers listenSpikers);
+  void sendFiles(std::string fileType = ".wav", int numberOfFiles = 42);
+  void connectToDsp(int numberOfDsp = 6);
+
+  speakers Listened;
+};
```

Comment:
Fix the following:
- Type Name capitalization
- Indentation relative to the `{ }`
- [ ] revise